spring-boot-project/spring-boot-tools/spring-boot-loader/src/test/java/org/springframework/boot/loader/archive/JarFileArchiveTests.java

```diff
@@ -131,8 +132,12 @@ public void unpackedLocationsFromSameArchiveShareSameParent() throws Exception {
 	public void zip64ArchivesAreHandledGracefully() throws IOException {
```

Comment:
This test name case should be renamed as it's not longer checking for an exception